modin/experimental/sql/__init__.py

```diff
@@ -0,0 +1,27 @@
+# Licensed to Modin Development Team under one or more contributor license agreements.
+# See the NOTICE file distributed with this work for additional information regarding
+# copyright ownership.  The Modin Development Team licenses this file to you under the
+# Apache License, Version 2.0 (the "License"); you may not use this file except in
+# compliance with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under
+# the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific language
+# governing permissions and limitations under the License.
+
+import warnings
+
+try:
+    import dfsql
+    from dfsql import sql_query
+    import dfsql.extensions
+except ImportError:
+    warnings.warn(
+        "Modin experimental sql interface requires dfsql to be installed."
+        " Run `pip install modin[sql]` to install it."
+    )
+    raise
+
+__all__ = ["dfsql", "sql_query"]
```

Comment:
```suggestion
__all__ = ["query"]
```

Reply:
I would rather leave `dfsql` exposed in case users want to use the full functonality of dfsql, like persisting tables

Reply:
Would you prefer `from modin.experimental.sql import dfsql` or `import dfsql`?

Reply:
Alright, it makes sense